test/multiapi/Expected/AcceptanceTests/Multiapi/multiapi/_operations_mixin.py

```diff
@@ -20,16 +20,20 @@ def test_one(
         **kwargs  # type: Any
     ):
         """TestOne should be in an SecondVersionOperationsMixin. Returns ModelTwo.
-    
-        :param id: An int parameter.
-        :type id: int
-        :param message: An optional string parameter.
-        :type message: str
-        :keyword callable cls: A custom type or function that will be passed the direct response
-        :return: ModelTwo or the result of cls(response)
-        :rtype: ~multiapi.v2.models.ModelTwo
-        :raises: ~azure.core.exceptions.HttpResponseError
-        """
+
+    :param id:
```

Comment:
Do you know why the spacing is weird, there's an extra new line between id and it's description. Also this entire docstring appears to be shifted to the left by 4 spaces